src/main/java/com/github/fge/jsonschema/format/common/DateTimeAttribute.java

```diff
@@ -42,13 +42,13 @@
     extends AbstractFormatAttribute
 {
     private static final List<String> FORMATS = ImmutableList.of(
-        "yyyy-MM-dd'T'HH:mm:ssZ", "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+        "yyyy-MM-dd'T'HH:mm:ssZ", "yyyy-MM-dd'T'HH:mm:ss.[0-9]{1,12}Z"
```

Comment:
Since it's just an error message why not just change to:
"yyyy-MM-dd'T'HH:mm:ss.S{1,12}Z"